content/events/2020-quebeccity/_index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Québec
+date: '2020-06-18'
+location:
+  countrycode: ca
+  country: Canada
+  city: Québec
+  url: https://goo.gl/maps/xTvcknrmbL5fQDEcA
+  venue: ENAP
+map: enap.PNG
+sched_slug: kubernetescommunitydayquebeccity
+social:
+  #twitter: 'https://twitter.com/Bouvet_3Y0Z'
```

Comment:
Did you create a Twitter handle for your event? If so, can you please update here? Thanks!

Reply:
Hi @KTan1226, we don't have such Twitter handle for the event. Thanks.